### PR TITLE
Update SCDF core to 1.3.0.BUILD-SNAPSHOT

### LINKED ISSUE #359: Upgrade to CF-deployer to 1.3 M2

### LINKED ISSUE #361: Upgrade SCDF-core to 1.3.0.BS
As a developer, I'd like to upgrade SCDF-core to 1.3.0.BUILD-SNAPSHOT. It is at 1.3.0.M2 right now.

### PR DESCRIPTION
**Pull request by sobychacko**
Fixes #361
Fixed #359

### CODE REVIEW

**Comment by sabbyanandan** [comment]
@sobychacko: It might be better to switch the CF-deployer to 1.3 BS as well. That way, we can run the CI plan against the OOM fixes included in the CF-deployer.

File: pom.xml [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
-		<version>1.3.0.M2</version>
+		<version>1.3.0.BUILD-SNAPSHOT</version>
 		<relativePath />
 	</parent>
 
 	<properties>
-		<spring-cloud-dataflow.version>1.3.0.M2</spring-cloud-dataflow.version>
-		<spring-cloud-deployer-cloudfoundry.version>1.3.0.M1</spring-cloud-deployer-cloudfoundry.version>
+		<spring-cloud-dataflow.version>1.3.0.BUILD-SNAPSHOT</spring-cloud-dataflow.version>
+		<spring-cloud-deployer-cloudfoundry.version>1.3.0.BUILD-SNAPSHOT</spring-cloud-deployer-cloudfoundry.version>
```
Comment (cppwfs) [inline pom.xml]: I may have misunderstood.  But isn't the  deployer to  be brought to .M2? 
i.e. https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry/issues/359

Comment (cppwfs) [reply pom.xml]: Also after discussion both dependencies will be BUILD-SNAPSHOT

**Comment by cppwfs** [comment]
LGTM

**Comment by cppwfs** [comment]
Squashed, merged.

**Pull request closed without merging**